framework/web/AssetConverter.php

```diff
@@ -33,10 +35,14 @@ class AssetConverter extends Component implements AssetConverterInterface
      *     'styl' => ['css', '@app/node_modules/bin/stylus < {from} > {to}'],
      * ]
      * ```
+     *
+     * Note: `Yii::getAlias()` can replace alias only at begin of command!
+     *
+     * @see https://sass-lang.com/documentation/cli/ SASS/SCSS
      */
     public $commands = [
         'less' => ['css', 'lessc {from} {to} --no-color --source-map'],
-        'scss' => ['css', 'sass {from} {to} --sourcemap'],
+        'scss' => ['css', 'sass --style=compressed --source-map-urls=relative {from} {to}'],
```

Comment:
`--source-map-urls=relative` seems to be default value so there's no need to specify it.

Reply:
Also, I don't think that we should default to compressed style.

Reply:
Also, it seems old syntax works with current sass without any issues.

Reply:
@samdark 
> Ruby Sass is [deprecated](https://sass-lang.com/ruby-sass), and we recommend you move to a different implementation

Ruby Sass not support new features, see diff between Dart and Ruby packages

Reply:
@samdark what's reason to generate uncompressed style? source map required

Reply:
For example, to read CSS generated during debugging?

Reply:
@samdark use source map to  display uncompressed css

Reply:
Alright.